packages/core/types/event.d.ts

```diff
@@ -1,16 +1,14 @@
 import Breadcrumb from "./breadcrumb";
+import * as common from "./common";
 
 declare class Event {
-  public static getStacktrace(
-    error: any,
-    errorFramesToSkip?: number,
-    generatedFramesToSkip?: number,
-  ): Stackframe[];
-
-  public static ensureEvent(
-    eventOrError: any,
-    errorFramesToSkip?: number,
-    generatedFramesToSkip?: number,
+  public static create(
```

Comment:
As per `_notify()` this isnt part of the public interface, but it is required for plugins to create unhandled errors.